pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -1134,6 +1127,11 @@ func TestLogicalStreamIngestionJobWithFallbackUDF(t *testing.T) {
 	RETURNS string
 	AS $$
 	BEGIN
+	SELECT crdb_internal.log((proposed).payload);
+        IF existing IS NULL THEN
+            RETURN 'accept_proposed';
+        END IF;
```

Comment:
i guess this is the only test that needed to be changed, now that udf is run as an applier instead of via conflict resolution?

Reply:
Yeah, the tests were all originally written for the applier mode except for this one.